src/main/java/org/fundacionJala/sfdc/commons/DriverManager.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionJala.sfdc.commons;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+/**
+ * DriverManager.java.
+ * Class that applies Singleton pattern to instance WebDriver only once.
+ */
+public final class DriverManager {
+    private static DriverManager driverManager;
+    private static WebDriver driver;
+
+    /**
+     * Constructor, private to apply singleton pattern.
+     */
+    private DriverManager() {
+        init();
+    }
+
+    /**
+     * Static method to get an class instance.
+     * @return instance.
+     */
+    public static DriverManager getInstance() {
+        if (driverManager == null) {
+            driverManager = new DriverManager();
+        }
+        return driverManager;
+    }
+
+    /**
+     * Initialize WebDriver object.
+     */
+    private void init() {
+        System.setProperty("webdriver.gecko.driver", "geckodriver.exe");
+        driver = new FirefoxDriver();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)